public/modules/custom/grants_admin_applications/src/Service/HandleDocumentsBatchService.php

```diff
@@ -158,7 +136,7 @@ public function process(int $batchId, array $docsToDelete, array &$context): voi
         $context['results']['failed_transaction_ids'][] = $transactionId;
         $context['results']['failed']++;
         $this->messenger->addError($e->getMessage());
-        $this->logger->get('grants_admin_applications')->error($e->getMessage());
+        $this->logger->error($e->getMessage());
```

Comment:
Is this a customer facing functionality? Printing the exception message might expose some sensitive information.